src/pages/index.tsx

```diff
@@ -34,18 +37,214 @@ export default function Home() {
+                    </Card>
+                  </a>
+                </Col>
+                <Col>
+                  <a
+                    title="Go to Kim Jong-il's page"
+                    href="/en/family_tree/Kim_Jong-il"
+                  >
+                    <Card>
+                      <ImgBox
+                        alt="Medium article"
+                        src="/examples/settings.png"
+                      />
+                      <Card.Body>
+                        Entitree is Multi-Label, and Multi-Language 👌
+                      </Card.Body>
+                    </Card>
+                  </a>
+                </Col>
+              </StyledRow>
+              <SubTitle>What they say about us:</SubTitle>
+              <StyledRow>
+                <Col>
+                  <Card>
+                    <Card.Body>
+                      <blockquote className="blockquote mb-0">
+                        <p>
+                          Wow. This is a really great website! It will
+                          definitely be useful for the research I do on a
+                          regular basis.
+                        </p>
+                        <footer className="blockquote-footer">
+                          <cite title="Source Title">UsefulCharts</cite>
+                        </footer>
+                      </blockquote>
+                    </Card.Body>
+                  </Card>
+                </Col>
+                <Col>
+                  <Card>
+                    <Card.Body>
+                      <blockquote className="blockquote mb-0">
+                        <p>
+                          I just learned about Entitree and I am really excited
+                          to see it!
+                        </p>
+                        <footer className="blockquote-footer">
+                          <cite title="Source Title">Wikidocumentaries</cite>
+                        </footer>
+                      </blockquote>
+                    </Card.Body>
+                  </Card>
+                </Col>
+              </StyledRow>
+              <StyledRow>
+                <Col>
+                  <Card>
+                    <Card.Body>
+                      <blockquote className="blockquote mb-0">
+                        <p>
+                          Congratulations to your really great tool! It is so
+                          much fun to surf through connected data with that.
+                        </p>
+                        <footer className="blockquote-footer">
+                          <cite title="Source Title">Garetien</cite>
+                        </footer>
+                      </blockquote>
+                    </Card.Body>
+                  </Card>
+                </Col>
+                <Col>
+                  <Card>
+                    <Card.Body>
+                      <blockquote className="blockquote mb-0">
+                        <p>Love it!</p>
```

Comment:
Anything Tony White said we can paste? shall we put the links to the website as well? google likes it

Reply:
Sure